### PR TITLE
feat: display auto sync status in application details view

### PR DESCRIPTION
**Pull request by alexef**
first minor change to display the auto sync status (enabled or not) in the application details page:
Screenshot 2022-11-05 at 12 11 01

see enhancement proposal: https://github.com/argoproj/argo-cd/issues/11102

Signed-off-by: Alex Eftimie <alex.eftimie@getyourguide.com>

Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [x] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

**Review by blakepettersson — APPROVED** [review]

File: ui/src/app/applications/components/application-status-panel/application-status-panel.tsx [hunk 1]
```diff
@@ -90,16 +90,17 @@ export const ApplicationStatusPanel = ({application, showOperation, showConditio
                         </div>
                         <div className='application-status-panel__item-value__revision'>{syncStatusMessage(application)}</div>
                     </div>
-                    <div className='application-status-panel__item-name'>
-                        {application.status && application.status.sync && application.status.sync.revision && (
+                    {application.status && application.status.sync && application.status.sync.revision && !application.spec.source.chart && (
+                        <div className='application-status-panel__item-name'>
```
Comment (alexef) [inline ui/src/app/applications/components/application-status-panel/application-status-panel.tsx:94]: this empty div makes no sense for Helm source (only for git)

**Review by rbreeze — CHANGES_REQUESTED** [review]
Hi Alex, thanks for the PR. I agree that we should have a more obvious place to indicate if Auto-Sync is enabled, but I'm not sure this is the best place for it; the Current Sync Status area can already be quite crowded: 

CleanShot 2022-11-10 at 08 25 13@2x

**Comment by alexef** [comment]
@rbreeze I see your point, but I can't think of a different place.

What if I change the area title, see mockup below? (Auto Sync Status vs Manual Sync Status)
Screenshot 2022-11-11 at 16 54 06
Screenshot 2022-11-11 at 16 54 19

**Comment by alexef** [comment]
@rbreeze after SIG UI yesterday, I assume we can give this approach another chance? I will follow up with the application list filter in a separate PR.

**Review by rbreeze — APPROVED** [review]

**Pull request merged**